File: 1288-remove-covered-intervals/1288-remove-covered-intervals.java
class Solution {
    public int removeCoveredIntervals(int[][] intervals) {
        int n=intervals.length;
        boolean arr[]=new boolean[n];
        for(int i=0; i<n; i++){
            for(int j=0; j<n; j++){
                if(j!=i && !arr[j]){
                    if(intervals[i][0]<=intervals[j][0] && intervals[j][1]<=intervals[i][1])arr[j]=true;
                }
            }
        }
        int count=0;
        for(int i=0; i<n; i++){
            if(arr[i])count++;
        }
        return intervals.length-count;
    }
}

File: 1288-remove-covered-intervals/1288-remove-covered-intervals.py
class Solution:
    def removeCoveredIntervals(self, intervals: List[List[int]]) -> int:
        lst=[]
        for i in range(0, len(intervals)):
            lst.append(False)
        
        for i in range(0,len(intervals)):
            for j in range(0, len(intervals)):
                if i!=j and lst[i]==False:
                    if intervals[i][0]<=intervals[j][0] and intervals[j][1]<=intervals[i][1]:
                        lst[j]=True
        count=0
        for i in lst:
            if i:
                count+=1
        return len(lst)-count
